js/iframe-coordinator/libs/client.ts

```diff
@@ -1,16 +1,11 @@
 import { Elm } from '../elm/Client.elm';
-import { PublicationHandler } from './types';
-
-interface ToastOptions {
-  title?: string;
-  custom?: any;
-}
+import { PublicationHandler, ToastingClient, ToastOptions } from './types';
 
 interface ClientConfigOptions {
   clientWindow?: Window;
 }
 
-class Client {
+class Client implements ToastingClient {
```

Comment:
Pulling in interfaces for features that clients can implement will keep the APIs consisent for client types (Traditional, Background, future types).

If we like this approach, I will add the NavigatingClient interface as well.